mobile/v1/redshift/sql-runner/sql/standard/03-optional-modules/01-app-errors/01-main/01-app-errors-limits.sql

```diff
@@ -0,0 +1,36 @@
+/*
+   Copyright 2021 Snowplow Analytics Ltd. All rights reserved.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+{{if eq (or .enabled false) true}}
+  -- Create a limit for this run - single row table.
+  DROP TABLE IF EXISTS {{.scratch_schema}}.mobile_app_error_run_limits{{.entropy}};
```

Comment:
Do we need a limits table specific to app errors? Could we have one that's generic to this layer of aggregation?

Reply:
I think having a limit table per optional module makes sense given the potential variance in event frequency & volume per event type

Reply:
I can see where you're coming from. IMO the benefit of the other way is that it offers a v simple way to interact with the model for users who need to join atomic data into their custom modules.

Not too pressed though, can see the benefit of both. :)

Reply:
Fair point. Would that not essentially be `{{.scratch_schema}}.{{.model}}_base_run_limits{{.entropy}}`?

Reply:
Hmm... Yes I think it would actually. Fair point, I'm happy with that.